### PR TITLE
feature: add read write of table metadata

### PR DESCRIPTION
**Pull request by sabinem**
- the metadata are stored in files `metadata.csv` that belong to the pipeline
- these csv files contain the columns `iten` , `type` (table or field), `language`, `description`
- the function `rw_metadata_table` prefills these files from the datasets, they can then be completed by the user
- if the file already exists, it will be read in unless the user explictely choses to overwrite

Later the files can be collected by another fucntion and added as metadata table in postgres

### CODE REVIEW

File: R/metadata_table.R
Comment (cmdoret) [inline R/metadata_table.R:1]: I feel like this function encapsulates two distinct jobs:
* Reading a [filled-in] metadata table
* Generating a template for manual fill-in

The former is a one-time operation (run manually by the person producing the dataset), while the latter may be rerun anytime the postgres metadata table must be regenerated.

**Comment by sabinem** [comment]
Update: the output was adapted according to https://github.com/statistikZH/statbotData/issues/50
Screenshot 2023-08-09 at 15 51 43

**Review by cmdoret — COMMENTED** [review]
The schema of the table looks good !
One comment - I just realized there's one thing I'm not sure I understand: In  `metadata_table_columns$data_type`, what determines whether we put `text` or `numeric`?

For example if I have a column with 120 categories, is it categorical?

**Comment by cmdoret** [comment]
Yes, thank you @sabinem !

**Review by cmdoret — APPROVED** [review]

**Pull request merged**